keyboards/dztech/dz60rgb/keymaps/MintCollie/config.h

```diff
@@ -0,0 +1,3 @@
+#define ENABLE_RGB_MATRIX_PIXEL_RAIN
```

Comment:
You need an include guard here: 
```suggestion
#pragma once

#define ENABLE_RGB_MATRIX_PIXEL_RAIN
```

As well as a license header.